package.json

```diff
@@ -36,7 +36,6 @@
     "@18f/eslint-plugin-identity": "^2.0.0",
     "@18f/identity-normalize-yaml": "^1.0.0",
     "@axe-core/puppeteer": "^4.5.1",
-    "@babel/eslint-plugin": "^7.14.5",
```

Comment:
Babel plugin was previously for `@18f/identity-eslint-plugin` compatibility, but this was removed in favor of modern syntax support:

https://github.com/18F/identity-idp/blob/main/app/javascript/packages/eslint-plugin/CHANGELOG.md#v200-2022-03-14